Gopkg.toml

```diff
@@ -65,31 +65,35 @@ required = [
 
 [[override]]
   name = "github.com/go-chassis/go-archaius"
-  revision = "7194a1be3948765395419f2257d00a28e285b77c"
+  version = "v0.19.0"
 
 [[override]]
   name = "github.com/kubeedge/beehive"
   revision = "06e0cfa222f7190d54740dd11f0fc1bc9e8085cb"
 
 [[override]]
   name = "k8s.io/code-generator"
-  version = "kubernetes-1.13.2"
+  version = "kubernetes-1.15.0"
 
 [[override]]
   name = "github.com/astaxie/beego"
   version = "v1.9.2"
 
+[[override]]
+  name = "github.com/containernetworking/cni"
+  version = "v0.6.0"
```

Comment:
does edgmesh work fine with this change ?

Reply:
Actually I was not test edgemesh based on this change.
The k8s v1.15 is using cni v0.6.0
https://github.com/kubernetes/kubernetes/blob/release-1.15/go.mod#L38
Hey @rohitsardesai83 , are you worring about this pr?
https://github.com/kubeedge/kubeedge/pull/893
Firstly let's bump into k8s v1.15, and then we can make more improvements.
Since there are lots of changed vendor codes, there are some risks for kubeedge v1.1.
So I guess it is better to update vendors earlier, and find the issues we need to fix.
How about that?